### PR TITLE
Metrics Webapp

### PR DESCRIPTION
**Pull request by shyusu4**
## PR Description

- Used React components.
- Used React props.
- Used React Router.
- Connected React and Redux.
- Handled events in a React app.
- Wrote integration tests with a React testing library.
- Used styles in a React app.
- Used React life cycle methods.
- Used store, actions, and reducers in React.

[Live Demo](https://634a40c19f59c75a5997a054--inquisitive-kheer-dffc53.netlify.app/)
[Video Link](https://www.loom.com/share/d3e35d0016024efa97e73b981d716631)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "f",
+  "version": "0.1.0",
+  "private": true,
+  "type": "module",
```
Comment (mirouhml) [inline package.json:5]: - [ ] Kindly remove `"type": "module",` what you're building is an app and not a module, this caused some issues for me I had to remove it to be able to run the project.

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @shyusu4, 

Good job so far!

There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Requested changes
_Check the comments under the review._

- [ ] Kindly record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. The content of the video **must** include:

  - a description of the project.
  - a demo of the project features.
  - you should also highlight some interesting piece of code or something you built that you are very proud of.

  For recording the video you can use tools like [Loom](https://www.loom.com/) that let you share a private link to the recording, and configure a shot that shows your computer screen and your face at the same time in a small picture, and then, include the link in your PR description, please.

- [ ] Please check your deployed app on Netlify because it isn't working for me. 😃
- [ ] Also, please add some integration tests using React Testing Library, as well as some unit tests for your components, as I was only able to find a single test for the redux store.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/__tests__/Home.test.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+// eslint-disable-next-line func-names
+window.matchMedia = window.matchMedia || function () {
+  return {
+    matches: false,
+    addListener() {},
+    removeListener() {},
+  };
+};
+
+describe('Home', () => {
```
Comment (OLIPLICHE) [inline src/__tests__/Home.test.jsx:17]: # Well done 🥇 
Capture

**Review by OLIPLICHE — APPROVED** [review]
Hi @shyusu4, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Highlights
- Well-recorded video👏
- README file is descriptive 👏
- The project is deployed👏 👏
- Test PASS👏


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/Home.test.jsx (hunk 2 above)
Comment (shyusu4) [reply src/__tests__/Home.test.jsx:17]: Thank you! 😄

**Pull request merged**